### PR TITLE
Fixed the incorrect README

### PR DESCRIPTION
**Pull request by spearmin10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Fixed the incorrect README. It was described on the `SetWithTemplate`.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FiltersAndTransformers/Scripts/ConcatFormat/README.md [hunk 1]
```diff
@@ -1,187 +1,87 @@
...
 
 ## Getting Started
 ---
-The script builds a text from a template text which includes variables such as:
-This is a test message for ${user_name}.
-
-The template will be formatted to `This is a test message for John Doe.` by replacing variable parameters.
-
-By default, a variable name starts with `${` and ends with `}` . You can change the start marker and end marker by specifying the `variable_markers` parameter.
+The transformer concatenates prefix and suffix which supports DT expressions to the string.
```
Comment (gal-forer) [inline Packs/FiltersAndTransformers/Scripts/ConcatFormat/README.md:32]: Hi, thanks for the donation, why did you reduced the description of the script ? The old one was more extensive, can you please elaborate or use the old one ?

File: Packs/FiltersAndTransformers/Scripts/ConcatFormat/README.md [hunk 2]
```diff
@@ -1,187 +1,87 @@
...
-!SetWithTemplate key=out template=${lists.Template} variable_markers=$
-```
-
-#### Lists Library
-Template:
-```
-My name is $first_name $last_name.
-```
+#### Parameters
+| **Argument Name** | **Value** | **Note** |
```
Comment (gal-forer) [inline Packs/FiltersAndTransformers/Scripts/ConcatFormat/README.md:40]: Can you please add the missing fields ? There are no value or note for some of the fields

File: Packs/FiltersAndTransformers/Scripts/ConcatFormat/README.md [hunk 3]
```diff
@@ -1,187 +1,87 @@
...
-```
+### Build an email address by adding a user ID to the domain
 
-#### Lists Library
-Template:
-```
-My name is ${first_name} ${last_name}.
-```
+#### Parameters
+| **Argument Name** | **Value** | **Note** |
```
Comment (gal-forer) [inline Packs/FiltersAndTransformers/Scripts/ConcatFormat/README.md:66]: Can you please add the missing fields ? There are no value or note for some of the fields

**Review by gal-forer — CHANGES_REQUESTED** [review]

**Comment by spearmin10** [comment]
@gal-forer The old one is the different README. so please don't compare with it.
The old one is the README for `SetWithTemplate`.
I've contributed `ConcatFormat` with `SetWithTemplate` before, but I've mistaken to copy the README for the script.
The README of the PR is the original one for it.

https://github.com/demisto/content/blob/c095f4b31d9c37788999e480a51c51889bcdbd35/Packs/CommonScripts/Scripts/SetWithTemplate/README.md

**Comment by gal-forer** [comment]
> @gal-forer The old one is the different README. so please don't compare with it. The old one is the README for `SetWithTemplate`. I've contributed `ConcatFormat` with `SetWithTemplate` before, but I've mistaken to copy the README for the script. The README of the PR is the original one for it.
> 
> https://github.com/demisto/content/blob/c095f4b31d9c37788999e480a51c51889bcdbd35/Packs/CommonScripts/Scripts/SetWithTemplate/README.md

Ok, but can you please add values for the examples @spearmin10

**Comment by spearmin10** [comment]
@gal-forer OK, I've updated it.

**Review by gal-forer — APPROVED** [review]

**Pull request merged**